content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -82,6 +82,7 @@ different Kubernetes components.
 | `CSIStorageCapacity` | `false` | Alpha | 1.19 | |
 | `CSIVolumeFSGroupPolicy` | `false` | Alpha | 1.19 | |
 | `ConfigurableFSGroupPolicy` | `false` | Alpha | 1.18 | |
+| `CronJobControllerV2` | `false` | Alpha | 1.20 | |
```

Comment:
Please also add a description for the feature gate (later in the page).

Reply:
Done.